tutorials/workflow/python/monitor-pattern/monitor/monitor_workflow.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass
+from datetime import timedelta
+import random
+import dapr.ext.workflow as wf
+
+wf_runtime = wf.WorkflowRuntime()
+
+@dataclass
+class Status:
+    isReady: bool
+
+@wf_runtime.workflow(name='monitor_workflow')
+def monitor_workflow(ctx: wf.DaprWorkflowContext, counter: int):
+    status = yield ctx.call_activity(check_status, input=counter)
+
+    if not status.isReady:
+        yield ctx.create_timer(fire_at=timedelta(seconds=2))
+        counter += 1
+        yield ctx.continue_as_new(counter)
```

Comment:
```suggestion
        yield ctx.continue_as_new(counter + 1)
```

You could just increment the counter directly here.